### DISCUSSION TITLE
Boolean Custom Fields

### DISCUSSION THREAD

**Discussion by YADOAK** [Courseplay/Courseplay_FS22#1974] [Q&A] [answered]
I have created a custom field (CP-1) on grass land (in Griffin, Indiana) but Courseplay won't recognise the pond. I have created a second custom field (CP-2) around the pond but i can't see how to subtract CP-2 from CP-1 to have Courseplay avoid the pond. Can this be done?

<img width="808" alt="Screenshot 2022-10-05 093402" src="https://user-images.githubusercontent.com/115046593/193942808-bb1d4590-db9a-4b74-8e4f-062b328b15b9.png">

**Comment by Tensuko** [comment, marked as answer]
Islands are not supported for custom fields. It won't work.

**Comment by Redster999** [comment]
How about the road goes to and around the pond here, then create the field like a horseshoe? Should then be ok

**Reply by Tensuko** [reply]
I would rather plow the field and make a real grass field (not drawed grass) if I want to work on it more often.
Maybe plow the outer border and the island.

**Reply by Redster999** [reply]
Yes! That would work better ;)
